src/SFA.DAS.Tools.Servicebus.Support.Core/Queue/QueueMessage.cs

```diff
@@ -29,6 +29,7 @@ public class QueueMessage
         public string ExceptionType { get; set; }
         [Newtonsoft.Json.JsonIgnore]
         public IEnumerable<KeyValuePair<string, object>> UserProperties => OriginalMessage.UserProperties.OrderBy((x => x.Key));
-
+        [JsonProperty("type")]
+        public string Type { get; set; } = "message";
```

Comment:
How does having different message types in the same collection affect performance of the DB queries?

Reply:
@DannyGB From my understanding there are no performance implications. It is one of the recommended ways to create relationship. I will add type to index creation.